### ISSUE TITLE
Update README.md to explain the features of this lib

### ISSUE THREAD

**Issue by lwouis** [joffrey-bion/csv-utils#1]
The readme says that this lib helps "handle CSV files easily.". It would be more helpful to list some features instead of broadly saying that it makes handling CSV files easily :p

**Comment by joffrey-bion** [comment]
Yes, obvioulsy, this is on my to-to list about cleaning my GitHub repositories ;-)

But before that, I need to actually update the code itself. It was written a long time ago and could really be improved, especially regarding encoding and locale (for number thousands separator and decimal point).
